src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -625,9 +631,37 @@ private async Task SendStreamItems<T>(string streamId, ChannelReader<T> reader,
             await SendWithLock(CompletionMessage.WithError(streamId, responseError));
         }
 
+#if NETCOREAPP3_0
+        // this is called via reflection using the `_sendStreamItems` field
+        private async Task SendIAsyncEnumerableStreamItems<T>(string streamId, IAsyncEnumerable<T> stream, CancellationToken token)
+        {
+            Log.StartingStream(_logger, streamId);
+
+            var combinedToken = CancellationTokenSource.CreateLinkedTokenSource(_uploadStreamToken, token).Token;
```

Comment:
It doesn't look like this combinedToken is being used yet. I'll review more once this is further along.